### ISSUE TITLE
AI studio V100训练速度比1080ti慢近3倍

### ISSUE THREAD

**Issue by qtz980805** [PaddlePaddle/Paddle#41244]
1）PaddlePaddle版本：2.2.2
    2）GPU：本地一块1080ti（11G）；AI studio notebook 一块TeslaV100 （16G/32G情况一样）
    3）本地1080ti，CUDA10.2，cudnn8.2；AI studio notebook TeslaV100， CUDA10.1，cudnn7.5；

本地一次iteration 2s左右，AI studio 一次iteration 5.5s左右。
请问可能的问题是什么？

在AI studio print了训练各个阶段的用时，分两种情况：
1）当某次迭代使用visualdl summary的时候，summary用时占大头（网络forward用时0.4s左右，总用时5.5s左右）
2）当某次迭代不使用visualdl summary的时候，网络forward用时占大头（网络forward用时3.8s左右，总用时还是5.5s左右）

代码段如下：
        for batch_idx, sample in enumerate(TrainImgLoader):
            start_time = time.time()

            outputs = model(sample["imgs"], sample["proj_matrices"], sample["depth_values"])
            #记录forward时间
            print(time.time() - start_time)
            loss, depth_loss = model_loss(outputs, sample["depth"], sample["mask"], dlossw=[float(e) for e in args.dlossw.split(",") if e])

            lr_scheduler.step()
            optimizer.clear_grad()
            loss.backward()
            optimizer.step()

            global_step = len(TrainImgLoader) * epoch_idx + batch_idx
            do_summary = global_step % args.summary_freq == 0
            if do_summary:
                depth_est = outputs["depth"]
                depth_gt = sample["depth"]["stage3"]
                mask = sample["mask"]["stage3"]

                log_writer.add_scalar(tag="train/loss", step=global_step, value=loss.item())
                log_writer.add_scalar(tag="train/depth_loss", step=global_step, value=depth_loss.item())
                log_writer.add_scalar(tag="train/abs_depth_error", step=global_step, value=AbsDepthError_metrics(depth_est, depth_gt, mask > 0.5))
                log_writer.add_scalar(tag="train/thres2mm_error", step=global_step, value=Thres_metrics(depth_est, depth_gt, mask > 0.5, 2))

                log_writer.add_image(tag="train/reference_img", img=sample["imgs"][0, 0].numpy(), step=global_step, dataformats="CHW")
                log_writer.add_image(tag="train/depth_full", img=depth_est[0].numpy(), step=global_step, dataformats="HW")
                log_writer.add_image(tag="train/depth_masked", img=(depth_est[0] * mask[0]).numpy(), step=global_step, dataformats="HW")
                log_writer.add_image(tag="train/depth_gt", img=depth_gt[0].numpy(), step=global_step, dataformats="HW")
                #记录一次迭代的总时间                                     
                print(
                    "Epoch {}/{}, Iter {}/{}, lr {:.6f}, train loss = {:.3f}, depth loss = {:.3f}, time = {:.3f}".format(
                        epoch_idx, args.epochs, batch_idx, len(TrainImgLoader),
                        optimizer.get_lr(), loss.item(), depth_loss.item(),
                        time.time() - start_time))

**Comment by jim19930609** [comment]
您好，问题已收到，正在排查中。请问可以提供一下您的AI Studio项目id，方便我们复现问题嘛？

**Comment by jim19930609** [comment]
另外也麻烦提供一下本地机器的CPU和内存配置，感谢！

**Comment by qtz980805** [comment]
好的，我正在把项目弄成公开的，请稍等下

**Comment by qtz980805** [comment]
项目链接：https://aistudio.baidu.com/aistudio/projectdetail/3562510?contributionType=1
本地服务器CPU：i9-9900K@3.60GHz 8核 32G内存

**Comment by qtz980805** [comment]
请问有找到问题吗？

**Comment by qtz980805** [comment]
请问有找到问题吗？或者您提供一个排查的方向,我自己调试？

**Comment by qtz980805** [comment]
解决了，是AI studio中cudnn版本较低的问题，实验发现使用cudnn8.0.4及以上的版本速度就快了
